apps/frontend/src/components/CreateRequestModal/CreateRequestModal.tsx

```diff
@@ -0,0 +1,47 @@
+import { Modal } from "react-bootstrap";
+import { Form } from 'react-router-dom';
+import styles from './CreateRequestModal.module.css';
+
+interface Props {
+  show: boolean,
+  handleClose: () => void,
+}
+
+export default function CreateRequestModal({ show, handleClose }: Props) {
+  function handleSubmit() {
+    handleClose();
+  }
+
+  return (
+    <Modal show={show} onHide={handleClose} animation={true} backdrop="static" centered>
+      <Modal.Header closeButton>
+        <Modal.Title>Create a request</Modal.Title>
+      </Modal.Header>
+      <Modal.Body>
+        <Form method='post' className={styles.form} name='new-request-form'>
+          <label className={styles.label} htmlFor='title'>Title:</label>
+          <input className={styles.input} type='text' name='title' id='title' required minLength={4}></input>
+
+          <label className={styles.label} htmlFor='content'>Content:</label>
+          <textarea className={styles.textarea} name='content' id='content' required></textarea>
+
+          <div className={styles.buttonContainer}>
+            <input className={styles.submit} type='submit' value='Submit' 
+            onClick={handleSubmit}></input>
+            <button
+              className={styles.button}
+              onClick={
```

Comment:
This closes the modal even if the form isn't successfully submitted, e.g. there's info missing.

Reply:
I need to create a separate issue out of it. We need to update the entire component, including its design.